metloom/pointdata/cdec.py

```diff
@@ -304,7 +302,7 @@ def get_daily_data(
         https://cdec.water.ca.gov/dynamicapp/req/JSONDataServlet?
         Stations=TNY&SensorNums=3&dur_code=D&Start=2021-05-16&End=2021-05-16
         """
-        return self._get_data(start_date, end_date, variables, ["D", "H", "E"])
+        return self._get_data(start_date, end_date, variables, ["D", "h", "E"])
```

Comment:
I think "H" might be an api thing on cdec side, not sure if 'h' works or not for that

Reply:
Nevermind, seems like it works!